### PR TITLE
Portal fantasy subnet info

### PR DESCRIPTION
**Pull request by Zaggo34**

### CODE REVIEW

**Comment by aaronbuchwald** [comment]
Can you sign all of the commits on this PR?

**Comment by Zaggo34** [comment]
> Can you sign all of the commits on this PR?

Hi @aaronbuchwald, I've now signed the commits.

**Review by aaronbuchwald — APPROVED** [review]
LGTM

File: networks/README.md [hunk 1]
```diff
@@ -11,14 +11,16 @@ WARNING: this information is provided by the community. Please exercise caution
...
 
 ## Mainnet
 
-| Chain ID | Network |                 Name |
-| :------- | ------: | -------------------: |
-| 53935    | Mainnet | DeFi Kingdoms Subnet |
+| Chain ID | Network |                  Name |
+| :------- | ------: | --------------------: |
+| 53935    | Mainnet |  DeFi Kingdoms Subnet |
+| 909      | Mainnet | Portal Fantasy Subnet |
```
[the remark this answers is not in the document]
Comment (Zaggo34) [reply networks/README.md:26]: Hey @ceyonur - we don't have our mainnet subnet up and running yet. So the `genesis.json` and contract details are not available. What do you recommend?

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**